### ISSUE TITLE
Terminal command failed audio cue never played when screen reader mode is enabled

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#39656]
Testing https://github-com/microsoft/vscode/-/205384

1. Have screen reader mode enabled
2. Enable terminal command failed audio cue
3. Run a command that fails like `asdf`
4. :bug: no sound is played
"accessibility.signals.terminalCommandFailed": {
    "sound": "on"
}
